cmd/authsvc/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:alpine AS build
```

Comment:
As @brenordr pointed out in one of my PRs, it might be better to specify which version of golang alpine is being used

Reply:
Can you show me how to do this?

Reply:
How is it done?